backend/src/crd/pkg/apis/viewer/register.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package viewer contains types that define the Viewer CRD for Kubeflow
+// Pipelines.
+package viewer
+
+const (
+	Kind      string = "Viewer"
```

Comment:
Yeah, I agree, it's not a great name. I thought of viewer since it's meant to proxy views of another webapp through ambassador that we launch and control. I can't think of another one right now, and I'm open to suggestions :-)